cypress/integration/unit/document-version.spec.js

```diff
@@ -52,6 +52,17 @@ context('Tests for KAS-1076', () => {
     cy.get('.vlc-scroll-wrapper__body').within(() => {
       cy.get('.vlc-document-card').as('docCards').should('have.length', 22);
     });
+
+    const linkedDocumentsNames = files.slice(9,14).map(file => file.newFileName);
+    const linkedDocumentTypes = files.slice(9,14).map(file => file.fileType);
+
+
+    cy.addLinkedDocumentToAgendaItem(linkedDocumentsNames);
+    cy.get(document.linkeddocumentTypeLabel).eq(0).contains(linkedDocumentTypes[0]);
```

Comment:
Een [lusje](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for) lijkt hier op z'n plaats? (baseer je op de lengte van de array!)